### ISSUE TITLE
Should we include references to Delft Stack?

### ISSUE THREAD

**Issue by manuGil** [TU-Delft-DCC/TU-Delft-DCC.github.io#53]
This is a website built by TU Delft with very handy tutorials, maybe to the resources page
https://www.delftstack.com/

**Comment by mwakok** [comment]
Yes, I have used that resource before as well. In general, I would say that if you find it useful, you can just add it to the resources. Although, it seems delftstack has decided to incorporate advertising on their website, which makes me less likely to use it 😒. 

We should of course think of a ways to categorize and curate this resources list (maybe assigning a maintainer). If the list grows very long, it might not be obvious which reference is best to use. For example, we could use a similar system as used in some journals to highlight certain reference we would highly recommend (for example the Turing guides) with symbols ** and *. 

In any case, it would be useful to review the entire list at some point.
